api.md

```diff
@@ -0,0 +1,168 @@
+## API
+
+All requests can be authenticated by a signed cookie with the session ID.
+
+### POST /session
+Start a new observation session.
+
+Request:
+```
+{
+  name: 'name',
+  observation_method: 'focal_animal',
+  focal_animal_name: 'tarzan',
+  focal_behavior_id: null,
+  session_interval_seconds: '100',
+  session_interval_duration: '15000',
+ 
+  scheme_id: 1234,
+  notes: 'thing',
+
+  // extra metadata
+  location: {
+    lat, long
+  }
+}
```

Comment:
For the front end, we've been talking about caching the observations for a session until we're ready to push it all up in a single payload so we don't need to make a bunch of individual round trips. Given that this app will be run on a cellular connection in a remote area, the potential for request failures is high, and tracking/retrying a single request failure is more reliable than N+1.

@tpinecone @andrewpurcell Thoughts?